From 6ede6d8585cb92a875d80991237c7f4b3f237c27 Mon Sep 17 00:00:00 2001
From: Saleem Abdulrasool <compnerd@compnerd.org>
Date: Fri, 17 Jun 2016 18:48:45 +0000
Subject: [PATCH] config.sub: recognise LLVM canonicalised windows triples

These are the various windows triples that are used by LLVM, ensure that we
recognise them properly:

  windows-gnu: MinGW
  windows-cygnus: Cygwin
  windows-msvc: MS ABI
  windows-itanium: MS ABI C + Itanium C++
---
 config.sub | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/config.sub b/config.sub
index eccd218..06b7ccc 100755
--- a/config.sub
+++ b/config.sub
@@ -118,7 +118,8 @@ case $maybe_os in
   linux-musl* | linux-uclibc* | uclinux-uclibc* | uclinux-gnu* | kfreebsd*-gnu* | \
   knetbsd*-gnu* | netbsd*-gnu* | netbsd*-eabi* | \
   kopensolaris*-gnu* | cloudabi*-eabi* | \
-  storm-chaos* | os2-emx* | rtmk-nova*)
+  storm-chaos* | os2-emx* | rtmk-nova* | \
+  windows-gnu* | windows-cygnus* | windows-msvc* | windows-itanium*)
     os=-$maybe_os
     basic_machine=`echo $1 | sed 's/^\(.*\)-\([^-]*-[^-]*\)$/\1/'`
     ;;
-- 
2.8.1

